web/src/js/utils.js

```diff
@@ -105,3 +105,17 @@ fetchApi.put = (url, json, options) => fetchApi(
         ...options
     }
 )
+
+
+export function getDiff(obj1, obj2) {
+    let result = {...obj2};
+    _.forIn(obj1, (value, key) => {
+    	  if(_.isEqual(obj2[key], obj1[key]))
+            result[key] = undefined;
+        else if(typeof Array.isArray(obj2[key]) && Array.isArray(obj2[key]))
+            result[key] = {...obj2[key]};
```

Comment:
i know, but the problem was that flow.response.headers is an array with lots of (tuple) arrays. if i generally replace all values which are equal we are not able to update the headers. e.g if you update `["host",  "abc.def"]` to  `["host",  "testf"]` will result into  `[undefined,  "testf"]`. thats the reason for the strange array handling